circuit_maintenance_parser/parsers/bso.py

```diff
@@ -0,0 +1,88 @@
+"""BSO parser."""
+
+import logging
+from typing import Dict
+
+from dateutil import parser
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+
+logger = logging.getLogger(__name__)
+
+
+class HtmlParserBSO1(Html):
+    """Notifications Parser for BSO maintenance notifications."""
+
+    def parse_html(self, soup):
+        """Execute parsing."""
+        data = {}
+        self.parse_tables(soup.find_all("table"), data)
+        return [data]
+
+    def parse_tables(self, tables: ResultSet, data: Dict):
+        """Parse Table tag."""
+        data["account"] = "Customer"  # No Account Name information in html/email details
```

Comment:
I would add a more clear information such as:
```suggestion
        data["account"] = "Customer info unavailable"  # No Account Name information in html/email details
```

Reply:
Thank you so much for suggesting. I would love to update if there are any other errors.

Reply:
only two small points:
* you should update the test results with the new "account" ("Customer info unavailable")
* if you could add typing to some parser methods missing, it would be great